resources/NBTest/NBTest_900_OptimizerDetailedSlow.py

```diff
@@ -444,6 +444,9 @@
 assert ti0.tknout == f"{T.USDC}"
 assert round(ti0.amtin, 8)  == 1214.45596849
 assert round(ti0.amtout, 8) == -1216.41933959
+if not ti0.error is None:
+    print(ti0)
+    print(ti0.error)
 assert ti0.error is None
```

Comment:
How about `assert ti0.error is None, f'{ti0.error}'` instead of those two `print` statements?

Reply:
it's somewhat nicer and cleaner but I actually want to print the whole object, not only the error

Reply:
So `assert ti0.error is None, f'{ti0}'`...